pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientOrganizationMapEntity.java

```diff
@@ -39,7 +39,7 @@
 	@JoinColumn(name = "organization_id", unique=true, nullable = true, insertable=false, updatable= false)
 	private OrganizationEntity organization;
 	
-	@Column(name = "organization_patient_record_id")
+	@Column(name = "organization_patient_id")
 	private String organizationPatientRecordId;
```

Comment:
Can you just change the variable name to match the column since we have Patient and PatientRecord? (So organizationPatientId instead of organizationPatientRecordId)